options.js

```
@@ -1,354 +1,251 @@
-    document.getElementById('vertex-max-tokens').value = options.vertexMaxTokens || '';
-    document.getElementById('vertex-context-window').value = options.vertexContextWindow || '';
-    document.getElementById('vertex-stream').value = options.vertexStream !== false ? 'true' : 'false';
-
-    document.getElementById('max-sessions').value = options.maxSessions || 3;
-
-    // Trigger change event to show the correct settings
-    apiTypeElement.dispatchEvent(new Event('change'));
-  });
+    'openRouterApiKey', 'openRouterModelId', 'openRouterMaxTokens', 'openRouterContextWindow', 'openRouterProviderOrder', 'openRouterAllowFallback',
+    'openaiApiKey', 'openaiModelId', 'openaiMaxTokens', 'openaiContextWindow', 'openaiBaseUrl'
+  ].forEach(key => setField(key, settings[key]));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)